main.go

```diff
@@ -201,7 +201,7 @@ func initProj(args []string) {
 }
 
 func installDeps() {
-	cmd := exec.Command("pio", "lib", "install")
+	cmd := exec.Command("~/.platformio/penv/bin/pio", "lib", "install")
 	cmd.Stdout = os.Stdout
```

Comment:
Thanks, can you check for Windows? 

The path looks like this on Windows `C:\Users\<user>\.platformio\penv\Scripts`

```
pioPath := "pio";
if runtime.GOOS != "windows" {
  pioPath = C:\Users\<user>\.platformio\penv\Scripts
}
```